### PR TITLE
[AArch64 cat] Tidying dependency related cat files

### PR DESCRIPTION
**Pull request by jalglave**

### CODE REVIEW

File: herd/libdir/aarch64show.cat [hunk 1]
```diff
@@ -8,3 +8,13 @@ show ctrl \ ctrlisb as ctrl
...
 
+(* Communication relations *)
+let rf-mem = rf \ rf-reg
+show rf-reg
+show rf-mem as rf
+show co|fr as ca
+
+(* Intrinsic order *)
+show iico_data as intrinsic
+show iico_ctrl as intrinsic
```
Comment (maranget) [inline herd/libdir/aarch64show.cat:19]: If I understand the idea (showing both iico's relation as intrinsic) , one would rather write:
```suggestion
-show iico_data as intrinsic
-show iico_ctrl as intrinsic
+show iico_data|iico_ctrl as intrinsic
```

**Comment by jalglave** [comment]
sure, let's do this - do you want me to or will you do it?

thanks,
jade

**Comment by jalglave** [comment]
Hi Luc,
It’s done now. Is it ready to merge into master?
Thanks,
Jade

**Comment by maranget** [comment]
Ok let's merge.

**Pull request merged**